File: src/main/java/com/github/avarabyeu/jashing/events/MeterEvent.java
package com.github.avarabyeu.jashing.events;

import com.github.avarabyeu.jashing.core.JashingEvent;

/**
 * Meter Event
 *
 * @author Andrei Varabyeu
 */
public class MeterEvent extends JashingEvent {

    private byte value;

    public MeterEvent(byte value) {
        this.value = value;
    }

    public MeterEvent(String title, byte value) {
        this.value = value;
        setTitle(title);
    }

    public byte getValue() {
        return value;
    }
}

File: jashing/src/main/kotlin/com/github/avarabyeu/jashing/events/MeterEvent.kt
package com.github.avarabyeu.jashing.events

import com.github.avarabyeu.jashing.core.JashingEvent

/**
 * Meter Event

 * @author Andrei Varabyeu
 */
class MeterEvent : JashingEvent {

    val value: Byte

    constructor(value: Byte) {
        this.value = value
    }

    constructor(title: String, value: Byte) {
        this.value = value
        this.title = title
    }
}
